projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.stats.execute.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.severing": "切割",
+  "tetra.stats.severing.tooltip": "攻击有§e%s%%§r概率使目标最大生命值§e-10%%§r，攻击伤害§e-5%%§r，持续§e60§r秒，此效果最多叠加§e%s§r层",
+  "tetra.stats.lunge": "猛冲",
+  "tetra.stats.lunge.tooltip": "按住右键蓄力，松开施展猛冲。猛冲击中实体时，会造成§e%s%%（%s）§f的伤害并使能力冷却时间§e-30%%§f\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.lunge.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.slam": "猛击",
+  "tetra.stats.slam.tooltip": "按住右键蓄力，松开释放强力猛击\n \n§7猛击地面会从撞击点发出冲击波，被冲击波击中的实体受到§e%s%%（%s）§7的伤害\n \n§7猛击的命中目标会受到§e%s%%（%s）§7的伤害并被强力击退，同时使能力冷却时间§e+50%%§7\n \n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.slam.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.puncture": "刺穿",
+  "tetra.stats.puncture.tooltip": "按住右键蓄力，松开发动可以刺穿目标盔甲的攻击，造成§e100%%（%s）§f的伤害并使目标护甲值§e-%s%%§f，持续§e%s§f秒\n \n§7如果刺穿一个护甲值少于§e6§7点或者已经被刺穿的目标，则会造成共计§e8§7点流血伤害，持续§e4§7秒\n \n§7能力冷却时间随目标护甲值增加而增加，每点护甲值§e+0.5§7秒\n \n§7蓄力时间：§e%s§7秒\n§7基础冷却时间：§e%s§7秒",
+  "tetra.stats.puncture.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.overpower": "压制",
+  "tetra.stats.overpower.tooltip": "按住右键蓄力，可以发动一次造成§e%s%%（%s）§f伤害但几乎不造成击退的攻击\n \n§7给予目标§e2§7层精疲力竭，给予攻击者§e1§7层。每层使移动速度§e-10%%§7，攻击速度§e-5%%§7。即使攻击失败，攻击者也会精疲力竭\n \n§7精疲力竭持续时间：§e%s§7秒\n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.overpower.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.reap": "收割",
+  "tetra.stats.reap.tooltip": "按住右键蓄力，松开发动横扫攻击，对范围内实体造成§e%s%%（%s）§f的伤害\n \n§7效果范围：§e%s§7格\n§7蓄力时间：§e%s§7秒\n§7冷却时间：§e%s§7秒",
+  "tetra.stats.reap.tooltip_extended": "蓄力时间和冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.pry_armor": "撬击",
+  "tetra.stats.pry_armor.tooltip": "右击1个实体造成§e50%%（%s）§f的伤害并使其护甲值§e-%s§f，持续§e%s§f秒，减甲效果可以叠加\n \n§7冷却时间：§e%s§7秒",
+  "tetra.stats.pry_armor.tooltip_extended": "冷却时间由物品的挥动速度决定，亡灵杀手等效果的额外伤害计入攻击的基础伤害中",
+  "tetra.stats.knockback": "击退",
+  "tetra.stats.knockback.tooltip": "击退距离§e+%s§r倍",
+  "tetra.stats.looting": "抢夺",
+  "tetra.stats.looting.tooltip": "增加击杀实体的掉落物",
+  "tetra.stats.fiery": "炽热",
+  "tetra.stats.fiery.tooltip": "攻击实体时使其着火§e%s§r秒",
+  "tetra.stats.sharpness": "锋利魔咒",
+  "tetra.stats.sharpness.tooltip": "通过魔咒将攻击伤害§e+%s§r（已计算在上方伤害数值内）",
+  "tetra.stats.smite": "亡灵杀手",
+  "tetra.stats.smite.tooltip": "在攻击亡灵生物时伤害§e+%s§r",
+  "tetra.stats.arthropod": "节肢杀手",
+  "tetra.stats.arthropod.tooltip": "在攻击节肢生物时伤害§e+%s§r，并附加1~§e%s§r秒的缓慢 IV效果",
+  "tetra.stats.sweeping": "横扫之刃",
+  "tetra.stats.sweeping.level": "横扫之刃伤害",
+  "tetra.stats.sweeping.efficiency": "横扫之刃范围",
+  "tetra.stats.sweeping.efficiency.tooltip": "横扫之刃范围§e+%s§r格",
+  "tetra.stats.sweeping.tooltip": "攻击生物时对附近半径§e%s§r格内的所有的敌人造成§e%s§r的武器伤害",
+  "tetra.stats.sweeping.tooltip_extended": "在空中或疾跑时横扫之刃不会触发",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Sweeping attacks do not trigger when attacking mid-air, while sprinting or when attacking during swing cooldown.**

```suggestion
  "tetra.stats.sweeping.tooltip_extended": "在空中或疾跑时横扫之刃不会触发",
```